### DISCUSSION TITLE
Can I download a specific version of a guide document?

### DISCUSSION THREAD

**Discussion by page-a** [rasbt/mlxtend#1027] [Q&A]
I'm so grateful for this package. Thank you for your dedication.

**Comment by rasbt** [comment]
Thanks for the kind compliment. Happy to hear that it's useful! Regarding the "guide document" versions, maybe the best way is to go the the list of released versions [here](https://github.com/rasbt/mlxtend/tags), download the respective zip file, and then render the documentation locally via `cd docs` and then `mkdocs serve`.
